### ISSUE TITLE
mcols(experiments(x)) are dropped by .harmonize()

### ISSUE THREAD

**Issue by lawremi** [waldronlab/MultiAssayExperiment#259]
`.harmonize()` should be careful to carry over the `mcols(experiments(x))`.

Like:
```
example("MultiAssayExperiment")
mcols(ExpList) <- DataFrame(foo=head(letters, length(ExpList)))
experiments(mae) <- ExpList
mcols(experiments(mae)) # NULL
```

And probably the `metadata()` also.

**Comment by LiNk-NY** [comment]
Thanks for the report Michael! @lawremi It should be fixed in `1.11.4`.

**Comment by lawremi** [comment]
Awesome, thanks.
